backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/RDFApi.java

```diff
@@ -54,25 +59,32 @@ private static int rdfForDatabase(Request request, Response response) throws IOE
 
   private static int rdfForSchema(Request request, Response response) throws IOException {
     Schema schema = getSchema(request);
-
-    RDFService rdf = new RDFService(request.url(), request.queryParams(FORMAT));
+    if (schema == null) {
+      throw new MolgenisException("Schema " + request.params("schema") + " was not found");
+    }
+    final String baseURL = extractBaseURL(request);
+    RDFService rdf = new RDFService(baseURL, RDF_API_LOCATION, request.queryParams(FORMAT));
     response.type(rdf.getMimeType());
 
     OutputStream outputStream = response.raw().getOutputStream();
-    rdf.describeAsRDF(outputStream, RDF_API_LOCATION, null, null, null, schema);
+    rdf.describeAsRDF(outputStream, null, null, null, schema);
     outputStream.flush();
     outputStream.close();
     return 200;
   }
 
   private static int rdfForTable(Request request, Response response) throws IOException {
     Table table = getTable(request);
-
-    RDFService rdf = new RDFService(request.url(), request.queryParams(FORMAT));
+    String rowId = null;
+    if (request.queryString() != null && !request.queryString().isBlank()) {
+      rowId = request.queryString();
```

Comment:
double check, we will never need other parameters in query string than id, right?

Reply:
At the moment we only support the id, but in the future we could implement filtering on other columns, but that would not conflict.
We should probably change the format to be based on content negotiation to avoid a possible collision and to be fully standards based (MarkW uses content negotiation), but I would then do that in a separate PR.
I don't foresee any other parameters in the RDF api at the moment.